css/gora.css

```diff
@@ -0,0 +1,540 @@
+body {
```

Comment:
Read about forms 

Inside `form` tag put all inputs
each input should have name and when you click on label - input should be in focus
when click on submit - form should be submitted! check in network tab in devtools
use button for submit button